packages/devtools_app/test/flutter/flame_chart_test.dart

```diff
@@ -12,14 +12,19 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter_widgets/flutter_widgets.dart';
 
 void main() {
+  const zoom = 1.0;
+
```

Comment:
https://api.flutter.dev/flutter/flutter_test/WidgetTester/sendKeyEvent.html

Can you add tests that use the new keyboard functionality?

Reply:
done